logsmanagement/stock_conf/logsmanagement.d.conf

```diff
@@ -0,0 +1,32 @@
+[global]
+    enabled = no
+    update every = 1
+    update timeout = 10
+    use log timestamp = auto
+    circular buffer max size MiB = 64
+    circular buffer drop logs if full = no
+    compression acceleration = 1
+    collected logs total chart enable = no
+    collected logs rate chart enable = yes
+
+[db]
+    db mode = none
+    # db dir = change to use non-default path
+    circular buffer flush to db = 6
+    disk space limit MiB = 500
+
+[forward input]
+    enabled = no
+    unix path = 
+    unix perm = 0644
+    listen = 0.0.0.0
+    port = 24224
```

Comment:
Please, add kernel log here disabled by default as we talked about.

Reply:
Hey @thiagoftsm , I will do that unless we can fix it by setting the correct capabilities.

Reply:
Today @ilyam8 gave us a suggestion associated with specific systemd files, but I also had issues on Qemu on a VM that does not have systemd.

Reply:
Fixed now.